### PR TITLE
feat: Update action with configuration options

### PR DESCRIPTION
**Pull request by cmitzel-ncino**
Updating the action to use configured git environment variables according to [vanilla semantic release](https://semantic-release.gitbook.io/semantic-release/usage/configuration) and [official git documentation](https://git-scm.com/book/en/v2/Git-Internals-Environment-Variables#_committing).

### CODE REVIEW

File: action.sh [hunk 1]
```diff
@@ -12,13 +12,16 @@ export PATH="${PATH}:/semantic-release/.venv/bin"
 # Change to configured directory
 cd "${INPUT_DIRECTORY}"
 
+export GIT_COMMITER_NAME: "${GIT_COMMITER_NAME:="github-actions"}"
+export GIT_COMMITER_EMAIL: "${GIT_COMMITER_EMAIL:="github-actions@github.com"}"
```
Comment (Kevmo92) [inline action.sh:16]:
```suggestion
export GIT_COMMITER_NAME="${GIT_COMMITER_NAME:="github-actions"}"
export GIT_COMMITER_EMAIL="${GIT_COMMITER_EMAIL:="github-actions@github.com"}"
```

**Review by Kevmo92 — COMMENTED** [review]
export command should be updated

**Review by Kevmo92 — APPROVED** [review]
Nice

https://media.giphy.com/media/yJFeycRK2DB4c/giphy.gif

**Review by relekang — CHANGES_REQUESTED** [review]
According to the [github actions documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations) this won't work:
>Any environment variables set in an env context defined at the workflow level in the caller workflow are not propagated to the called workflow.

This needs to be configured to with [action inputs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs)

**Comment by relekang** [comment]
And btw the documentation you are referring to is the node semantic release and not python semantic release. Our documentation is on https://python-semantic-release.readthedocs.io/en/latest/

**Comment by cmitzel-ncino** [comment]
@relekang does that look right now?

And yeah, I was under the impression that the projects were related in some way but I must have misunderstood. 

We're trying to just make sure that we can create a commit and release signed by our service user rather than using the general actions user.

File: action.sh [hunk 2]
```diff
@@ -12,13 +12,16 @@ export PATH="${PATH}:/semantic-release/.venv/bin"
 # Change to configured directory
 cd "${INPUT_DIRECTORY}"
 
+export GIT_COMMITER_NAME="${GIT_COMMITER_NAME:="github-actions"}"
```
Comment (relekang) [inline action.sh:15]:
```suggestion
export GIT_COMMITER_NAME="${INPUT_GIT_COMMITER_NAME:="github-actions"}"
```

File: action.sh [hunk 3]
```diff
@@ -12,13 +12,16 @@ export PATH="${PATH}:/semantic-release/.venv/bin"
...
+export GIT_COMMITER_EMAIL="${GIT_COMMITER_EMAIL:="github-actions@github.com"}"
```
Comment (relekang) [inline action.sh:16]:
```suggestion
export GIT_COMMITER_EMAIL="${INPUT_GIT_COMMITER_EMAIL:="github-actions@github.com"}"
```

File: action.sh (hunk 2 above)
Comment (cmitzel-ncino) [reply action.sh:15]: good catch, i merged in kevmo's changes wrong :oof:

**Review by relekang — APPROVED** [review]

**Review by Kevmo92 — APPROVED** [review]
Thanks!

**Comment by relekang** [comment]
Thanks @cmitzel-ncino 🙌

**Pull request merged**